tests/Container/ContainerTest.php

```diff
@@ -795,3 +911,13 @@ function containerTestInject(ContainerConcreteStub $stub, $default = 'taylor')
 {
     return func_get_args();
 }
+
+class ContainerTestContextInjectInstantiations implements IContainerContractStub
+{
+    public static $instantiations = 0;
```

Comment:
Does this work properly if you use the flag on phpunit that lets you run the tests multiple times?